### ISSUE TITLE
`peers` JSON-RPC method

### ISSUE THREAD

**Issue by aesedepece** [witnet/witnet-rust#312]
This should return something like:
```json
[
  {
    "address": "192.168.100.1:1234",
    "type": "inbound"
  }, {
    "address": "192.168.100.2:5678",
    "type": "outbound"
  }
]
```

**Comment by Tommytrg** [comment]
hey @csmoe! Are you still working on this issue?

**Comment by csmoe** [comment]
@Tommytrg really sorry, time is tough for me, feel free to do anything about this.

**Comment by tmpolaczyk** [comment]
Implemented in #1165
